docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.7"
+
+services:
+  interview_backend_coop_logistics:
+    build: .
+    container_name: interview_backend_coop_logistics
+    environment:
+      - APP_PORT=50051
+    tty: true
+    ports:
+      - "50051:50051"
+  techdocs:
+    build:
+      context: .
+      dockerfile: internal/techdocs/Dockerfile
```

Comment:
Why are we not using our common pattern of having this in `/docker-compose/`?

Reply:
Minimized "Coop specifics" so for participants would be a bit easier, could create questions like "why docker-compose folder with images docker images? etc"

We can move it to the root as well, not a big deal.